.dadew

```diff
@@ -8,7 +8,8 @@
         "ojdkbuild11",
         "maven-3.6.1",
         "msys2",
-        "nodejs",
+        "nodejs-14",
+        "nodejs-16",
```

Comment:
How do you validate that this actually works with Node.js 18?

Reply:
Manually.

Reply:
Should we make it part of our checks on CI?

Reply:
I wasn't aware, thanks for clarifying. Can you write down what went wrong? If it's complex, maybe as an issue (unless you have done so already, of course). [Node.js 16 is EoL in September 2023](https://github.com/nodejs/Release#release-schedule) and understanding what obstacles are there could help make a further transition smoother.